diff --git a/node_modules/@chainlink/ccip-js/dist/api.js b/node_modules/@chainlink/ccip-js/dist/api.js
index e5621e2..bbd0a7c 100644
--- a/node_modules/@chainlink/ccip-js/dist/api.js
+++ b/node_modules/@chainlink/ccip-js/dist/api.js
@@ -6,7 +6,7 @@ import OnRampABI_1_6 from './abi/OnRamp_1_6.json';
 import IERC20ABI from './abi/IERC20Metadata.json';
 import TokenPoolABI from './abi/TokenPool.json';
 import TokenAdminRegistryABI from './abi/TokenAdminRegistry.json';
-import { TRANSFER_STATUS_FROM_BLOCK_SHIFT, ExecutionStateChangedABI } from './config';
+import { TRANSFER_STATUS_FROM_BLOCK_SHIFT, ExecutionStateChangedABI } from './config.js';
 import { parseAbi } from 'viem';
 export { IERC20ABI };
 /**
